scripts/lib/setup-yum-repo

```diff
@@ -53,7 +53,7 @@ elif [ "$is_rhel" = true ]; then
     yum localinstall -y https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-latest-x86_64/pgdg-redhat10-10-2.noarch.rpm
     yum localinstall -y https://packages.groonga.org/centos/groonga-release-latest.noarch.rpm
 else
-    # TODO only fedora33 for now
+    # TODO only fedora34 for now
     PGVER=13
-    dnf install -y "https://download.postgresql.org/pub/repos/yum/reporpms/F-33-x86_64/pgdg-fedora-repo-latest.noarch.rpm"
+    dnf install -y "https://download.postgresql.org/pub/repos/yum/reporpms/F-34-x86_64/pgdg-fedora-repo-latest.noarch.rpm"
```

Comment:
I think it'd be better to make both Fedora 33 and 34 work, since fedora 33 hasn't reached EOL yet.

Reply:
Which variable must I assign to resolve the problem of 33/34 version conflicts?